src/Api/Controllers/AccountsController.cs

```diff
@@ -618,5 +618,38 @@ public async Task<string> GetEnterprisePortalSignInToken()
 
             return token;
         }
+        
+        [HttpGet("tax")]
+        [SelfHosted(NotSelfHostedOnly = true)]
+        public async Task<TaxInfoResponseModel> GetTaxInfo()
+        {
+            var user = await _userService.GetUserByPrincipalAsync(User);
+            if (user == null)
+            {
+                throw new UnauthorizedAccessException();
+            }
+
+            var taxInfo = await _paymentService.GetTaxInfoAsync(user);
+            return new TaxInfoResponseModel(taxInfo);
+        }
+        
+        [HttpPut("tax")]
+        [HttpPost("tax")]
```

Comment:
We don't really need these anymore. We used to POST everything too since our web vault had CORS issues back in the day. This has since been resolved with the proxy. If it's truly a PUT, just make it PUT only